### PR TITLE
Fixes building problem in node version 6

### PR DESCRIPTION
**Pull request by Wulf**

### CODE REVIEW

**Comment by nathanjsweet** [comment]
Do your changes break if the person is using and older version of node? Let's say 0.12.0?

File: src/post0.12.0/nodehun.cpp [hunk 1]
```diff
@@ -4,14 +4,14 @@ using namespace v8;
...
-void Nodehun::SpellDictionary::Init(Handle<Object> exports, Handle<Object> module) 
+void Nodehun::SpellDictionary::Init(Handle<Object> exports, Handle<Object> module)
 {
   Isolate* isolate = Isolate::GetCurrent();
   Local<FunctionTemplate> tpl = FunctionTemplate::New(isolate, New);
   tpl->SetClassName(String::NewFromUtf8(isolate, "NodehunDictionary"));
   tpl->InstanceTemplate()->SetInternalFieldCount(1);
   //static
-  NODE_SET_METHOD(tpl, "createNewNodehun" , createNewNodehun);    
+  NODE_SET_METHOD((v8::Local<v8::Template>) tpl, "createNewNodehun" , createNewNodehun);
```
Comment (nathanjsweet) [inline src/post0.12.0/nodehun.cpp]: Why this coercion?

Comment (Wulf) [inline src/post0.12.0/nodehun.cpp]: Please find the latest node headers [here](https://github.com/nodejs/node/blob/master/src/node.h).

The latest version of node ships with headers which overload `NODE_SET_METHOD` (see lines 238 and 251). Thus, the coercion was necessary in order to disambiguate between the two method signatures.

**Comment by Wulf** [comment]
I checked this with older pre-12.0, post-12.0 and also on `6.2.2`.
It built successfully, and the tests showed accurate results. 

And yes, it's just this casting change that was required to prevent breaking the builds on node versions over 5.

**Pull request merged**